### DISCUSSION TITLE
Script injection

### DISCUSSION THREAD

**Discussion by eddysilvamendes** [osTicket/osTicket#6714] [Q&A]
Hello. Can somenone help me pls.

I was creating a new tiket for my company. and i made a test with a script to replace the button from one of mine and redirected to a new page. and it work. this is a huge security problem, cuz i could use it for pushing for example. now i'm using version 15 but a try it on 17 and it work 2. how can i solve this.
<img width="744" alt="Screenshot 2024-06-10 at 12 01 06" src="https://github.com/osTicket/osTicket/assets/16749776/5d81079c-a7c3-440f-9bc7-49a4dc5d794a">

**Comment by JediKev** [comment]
@eddysilvamendes …huh?

Also, please review here for security reports:
-  https://github.com/osTicket/osTicket/security/policy

Cheers.
